rsyslog/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM ubuntu:22.04
```

Comment:
Is Ubuntu needed here? Is there an rsyslog package in alpine?

Reply:
seems to be an rsyslog pkg in alpine:3.17.3

Reply:
we dont need ubuntu. i do like ubuntu :)